boards/st/nucleo_n657x0_q/nucleo_n657x0_q_defconfig

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 STMicroelectronics
+
+# enable uart driver
```

Comment:
capitalise first letters and UART not uart

Reply:
Done